portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,7 +39,47 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
  
-  private List<String> comments;
```

Comment:
make comments into a list of Comment objects which store all the pertinent info regarding a comment. This is to make sure that the comments are displayed with the appropriate level of information on each recipe page